examples/analytics-vendors.amp.html

```diff
@@ -715,6 +716,30 @@
 </amp-analytics>
 <!-- End INFOnline example -->
 
+<!--
```

Comment:
Hello @infonline. Can I ask why we need the infonline_anonymous as a different vendor? 
Is it possible to set to anonymous mode via a var? For example
```
<amp-analytics type=infonline>
<script type="application/json">
{
  "vars": {
     "anonymous": true
   }
   ...
}
</script>
</amp-analytics>
```

Reply:
Thank you very much for reviewing the PR. Due to privacy protection requirements, 
we want to completely separate the previous solution from the new anonymous solution. 
We also want to avoid technical dependencies at this point.

Reply:
SGTM. We want to best support your use case. Given all vendor configs are loaded separately and lazily, we are good with having the anonymous solution registered as a standalone vendor.